### ISSUE TITLE
Left navigation accessibility issue

### ISSUE THREAD

**Issue by dajoturner** [ucsc/wcms-design-templates#269]
Len Burns issue. Basically, he's saying that the left navigation is coming in at the very bottom of the page, and that it doesn't have an anchor/heading associated to make it easy to find.

Setting this to for Rob to evaluate: @knice If this is a pain, I'd suggest setting it to Could Wait. Just needs you to take a look.

See: http://admissions-dev.ucsc.edu/campus-life/index.html

Len's text:
I am finding a pattern on a number of pages which use a navigation to the
left side of the page.  Since a screen reader reads text in a liniar order,
code order, these navs end up at the bottom of the page with no means of
realizing they are present.  The example from this page is:

“list of 5 items

Campus Life

Getting Involved

Support Services

Rush Week 2012

Our Student Profiles”

Reading the page sequentially, this is just after the main region, nearly at
the bottom of the page.  Were I not exploring, I would have missed it.  I
suggest these navs be anchored, perhaps by a heading, so they can be easily
located.
